clusterloader2/testing/density/config.json

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
Can we please use yamls instead of jsons?

They are much more concise (and a bit more readable in my opinion)

Reply:
Can't agree more :)

Reply:
Also, it'll probably make this file half the size.

Reply:
Converted to yaml.
IMHO, json looked better.